src/components/HowItWorks/Title.tsx

```diff
@@ -1,14 +1,17 @@
 import { HeaderText } from 'components/ui/Text'
 import { displayFrom, displayTo } from 'helpers/visibilityClassnames'
 
-export default function () {
+export default function ({ hasWallet }: { hasWallet?: boolean }) {
```

Comment:
`{ hasWallet?: boolean }` is used 2 times, worth extracting into separate file `interface`? If you think yes, let's do it

Reply:
We will change it in the nearest future so that we might keep it like this, without an interface